lib/hash.cc

```diff
@@ -66,22 +66,22 @@ hash4(const void *data, unsigned int size)
         break;
     case 7:
         HASH4;
-    /* [[fallthrough]] */
+        [[fallthrough]];
     case 6:
```

Comment:
Please note that these formatting improvements were not manual -- astyle appears to handle `[[fallthrough]]` attributes better than trailing `case` comments.